converter/tests/test_utils.py

```diff
@@ -212,6 +212,12 @@ def test_update_json_value_several_matches(self):
         for code in self.json_data["qualification"]["riskThreat"]:
             self.assertEqual(code["code"], 100)
 
+    @patch('builtins.print')
+    def test_update_json_value_no_match(self, mock_print):
+        update_json_value(self.json_data, "$.qualification.riskThreat[*].toto", 100)
+        mock_print.assert_not_called()
```

Comment:
Pourquoi passer par le print pour s'assurer qu'il n'y a pas eu d'exception ?
Pourquoi pas quelque chose du genre ?
```suggestion
    def test_update_json_value_no_match(self):
        try:
            update_json_value(self.json_data, "$.qualification.riskThreat[*].toto", 100)
        except Exception:
            self.fail("update_json_value raised an exception unexpectedly")
```

Reply:
modifié